README.md

```diff
@@ -1,6 +1,14 @@
-# Co-Coders API 
+# Co-Coders API
+
 [![Build Status](https://travis-ci.org/clean-code-projects/co-coders-api.svg?branch=master)](https://travis-ci.org/clean-code-projects/co-coders-api)
 
+## contribute
+

```

Comment:
Let's add a line about needing to replicate the folder structure with github.

Reply:
I don't know what's the difference but I actually don't need to follow the folder structure.
```sh
laptop:co-coders-api hwong$ go test ./...
?   	github.com/clean-code-projects/co-coders-api/internal/api	[no test files]
?   	github.com/clean-code-projects/co-coders-api/internal/co_coders/criteria[no test files]
ok  	github.com/clean-code-projects/co-coders-api/internal/co_coders/match	(cached)
?   	github.com/clean-code-projects/co-coders-api/internal/co_coders/user	[no test files]
laptop:co-coders-api hwong$ pwd
/Users/hwong/code/clean-code-projects/co-coders-api
```

Ran `mv clean-code-projects ccp` and it still works.

Reply:
Nice, go modules fixes those issues.